lib/cocoapods/installer/analyzer.rb

```diff
@@ -532,7 +532,7 @@ def resolve_dependencies
         end
 
         specs_by_target = nil
-        UI.section "Resolving dependencies of #{UI.path podfile.defined_in_file}" do
+        UI.section "Resolving dependencies of #{UI.path(podfile.defined_in_file) || 'Podfile'}" do
```

Comment:
This change seems unrelated? Also, shouldn't `podfile.defined_in_file` always have a value?